chrome/content/zotero/lookup.js

```diff
@@ -98,19 +99,36 @@ var Zotero_Lookup = new function () {
 			}
 			// Continue with other ids on failure
 			catch (e) {
+				let service = Object.keys(identifier)[0];
+				failedIDs.push(service + ': ' + identifier[service]);
```

Comment:
Technically this isn't localized — not all languages use a colon in this way. But it's probably fine.

Reply:
Ok. I wasn't really sure what to do here because there is no easy way to actually display what the user typed in and relate that to the parsed identifier. This would in theory only be DOI, ISBN, or arXiv so the colon seemed ok.